ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V100__Append_only_schema.sql

```diff
@@ -0,0 +1,753 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+
+---------------------------------------------------------------------------------------------------
+-- V100: Append-only schema
```

Comment:
just to make absolutely sure: my current understanding is that with this on main we will start doing migration tests, but we do not support append-only for production usage/data continuity yet, right? cc @mziolekda @rautenrieth-da 
This is important to know: we can still change this file as going onwards (some refactorings to the schema would be much better looking if we "squash" them to this sql script, instead of subsequent ones)